@kiva/kv-components/tests/unit/specs/components/KvWideLoanCard.spec.js

```diff
@@ -0,0 +1,63 @@
+import { render } from '@testing-library/vue';
+import { axe } from 'jest-axe';
+import KvWideLoanCard from '../../../../vue/KvWideLoanCard.vue';
+
+const nextWeek = new Date();
+nextWeek.setDate(new Date().getDate() + 7);
+const kvTrackFunction = () => { };
+
+const photoPath = 'https://www-kiva-org.freetls.fastly.net/img/';
+const loan = {
+	id: 1,
+	name: 'Alan',
+	geocode: {
+		city: 'Lyantonde',
+		state: 'Central Region',
+		country: {
+			isoCode: 'UG',
+			name: 'Uganda',
+			region: 'Africa',
+			__typename: 'Country',
+		},
+		__typename: 'Geocode',
+	},
+	image: { hash: '9673d0722a7675b9b8d11f90849d9b44' },
+	fundraisingPercent: 0.5,
+	unreservedAmount: '500.00',
+	use: 'to purchase heifers to increase headcount of cattle and sales of organic milk.',
+	status: 'fundraising',
+	loanAmount: '1000.00',
+	borrowerCount: 1,
+	activity: {
+		id: 61,
+		name: 'Dairy',
+		__typename: 'Activity',
+	},
+	sector: {
+		id: 1,
+		name: 'Agriculture',
+		__typename: 'Sector',
+	},
+	plannedExpirationDate: nextWeek.toISOString(),
+};
+
+describe('KvWideLoanCard', () => {
+	it('has no automated accessibility violations', async () => {
+		const { container } = render(KvWideLoanCard,
+			{
+				props: {
+					loanId: loan.id,
+					loan: {
+						...loan,
+						unreservedAmount: undefined,
+						fundraisingPercent: undefined,
+					},
+					kvTrackFunction,
+					photoPath,
+					externalLinks: true,
+				},
+			});
+		const results = await axe(container);
+		expect(results).toHaveNoViolations();
+	});
```

Comment:
Coule we add an additionall test or 2 that checks the results of your loanCard util? If we can test the rendering of a couple of those computed properties across vue2 and vue3 that would help verify the functionality.

Reply:
Added tests of the country display. Since that seem pretty useful. It tests `formattedLocation` `countryName` and `distributionModel` computed props